esmvalcore/cmor/_fixes/cmip6/gfdl_cm4.py

```diff
@@ -0,0 +1,64 @@
+"""Fixes for CESM2 model."""
+from ..fix import Fix
+from ..shared import (add_scalar_height_coord, add_scalar_height_coord_10,
+                      add_scalar_typeland_coord)
+
+
+class Tas(Fix):
+    """Fixes for tas."""
+
+    def fix_metadata(self, cubes):
+        """Add height (2m) coordinate.
+
+        Parameters
+        ----------
+        cube : iris.cube.CubeList
+
+        Returns
+        -------
+        iris.cube.Cube
+
+        """
+        cube = self.get_cube_from_list(cubes)
+        add_scalar_height_coord(cube)
+        return cubes
+
+
+class Uas(Fix):
+    """Fixes for tas."""
+
+    def fix_metadata(self, cubes):
+        """Add height (10m) coordinate.
+
+        Parameters
+        ----------
+        cube : iris.cube.CubeList
+
+        Returns
+        -------
+        iris.cube.Cube
+
+        """
+        cube = self.get_cube_from_list(cubes)
+        add_scalar_height_coord_10(cube)
```

Comment:
Instead of adding a new function, just call `add_scalar_height_coord(cube, 10.)`.